plugins/woocommerce/src/Blocks/BlockTypesController.php

```diff
@@ -474,6 +474,7 @@ protected function get_block_types() {
 			'ReviewsByProduct',
 			'RelatedProducts',
 			'ProductDetails',
+			'ProductDescription',
```

Comment:
I see that we are conditionally registering it in the `index.tsx` file already, but the file would still get enqueued.
Would it make sense to not instantiate the class either, as we are doing for the product details block, see items under this condition: `if ( Features::is_enabled( 'experimental-blocks' ) ) {`

Reply:
Ah, good catch. Sometimes I get confused between `ProductDetails` and `BlockifiedProductDetails`. My bad 😅 I will make the changes.